verilog/CST/verilog_tree_print.h

```diff
@@ -41,6 +44,29 @@ class VerilogPrettyPrinter : public verible::PrettyPrinter {
 void PrettyPrintVerilogTree(const verible::Symbol& root, absl::string_view base,
                             std::ostream* stream);
 
+class VerilogTreeToJsonConverter : public verible::SymbolVisitor {
```

Comment:
If this class never needs to be referenced directly/publicly, moved it into the .cc file as an implementation detail.  

I'd also be ok with adding this in a separate files, like `CST/verilog_json.h` `CST/verilog_json.cc`.  This keeps the dependencies thinner for the rest of the codebase that doesn't need JSON support.

Reply:
Done.